examples/mlm-metadata.yaml

```diff
@@ -0,0 +1,39 @@
+name: Unet
+architecture: Unet
+artifact_type: torch.export.save
+framework: torch
+framework_version: 2.7.0
+accelerator: cuda
+total_parameters: 1234567
+tasks: [semantic-segmentation]
+input:
+  - name: Imagery
+    bands: [red, blue, green]
+    input:
+      shape: [-1, 3, 512, 512]
+      dim_order: [batch, channel, height, width]
+      data_type: float32
+      res: 0.1
```

Comment:
While additional properties are allowed, this one is technically not part of MLM schema. Is this property required? If so, it is fine to leave it to make sure via tests that it is still allowed, but a comment would be needed to indicate that this is extra/outside of MLM scope.

Reply:
It is not so I can remove